pkg/linstorhelper/properties.go

```diff
@@ -0,0 +1,75 @@
+package linstorhelper
+
+import (
+	"encoding/json"
+	"sort"
+
+	linstor "github.com/LINBIT/golinstor"
+	lclient "github.com/LINBIT/golinstor/client"
+
+	"github.com/piraeusdatastore/piraeus-operator/v2/pkg/vars"
+)
+
+const (
+	LastApplyProperty = linstor.NamespcAuxiliary + "/" + vars.ApplyAnnotation
+	ManagedByProperty = linstor.NamespcAuxiliary + "/" + vars.ManagedByLabel
+)
+
+// GeneratePropertiesPatch returns the modification that need to be applied to update the current properties.
+func GeneratePropertiesPatch(current, expected map[string]string) *lclient.GenericPropsModify {
```

Comment:
"Patch" has quite a specific meaning in Kubernetes. This is, of course, similar, but it is not an actual Kubernetes patch. The name confused me when I first read it. Maybe use "Modify" like the linstor client? (Even though the grammar is a bit jarring.)

Reply:
What about `GeneratePropertiesModification`? Or `MakePropertiesModification`? I think both would be rather readable.

Reply:
Both sound good to me. Take your pick.